app/forms/course_details_form.rb

```diff
@@ -129,7 +129,7 @@ def attrs_from_course_age_range(course_age_range)
 
     attrs = {}
     if age_range.present? && !trainee.early_years_route?
-      attrs["#{age_range[:option]}_age_range".to_sym] = course_age_range.join(" to ")
+      attrs[:"#{age_range[:option]}_age_range"] = course_age_range.join(" to ")
```

Comment:
not for me to question rubocop, but i'm not convinced `:"whatevz"` is an improvement on `"whatevz".to_sym`